### PR TITLE
Use Documenter-generated output for logging demo.

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: docs/src/adv/logging.md [hunk 1]
```diff
@@ -17,40 +17,41 @@ import Term: install_term_logger
 install_term_logger()
 ```
 
-!!! note
-    We could not get Documenter.jl to show the output of `Term`'s logger. So we've added a screenshot at the bottom of this page!
```
Comment (fredrikekre) [inline docs/src/adv/logging.md:21]: Maybe _We_ can't, but _I_ can :slightly_smiling_face:

**Comment by FedeClaudi** [comment]
Brilliant! I suspected that there should be a way, but could not get it to work yesterday. Is there something similar for error messages?

Thank you so much for helping with Term!

**Comment by fredrikekre** [comment]
> Is there something similar for error messages?

Probably, you would have to put in `try`-`catch` and invoke `showerror` yourself. I have not tried it out though.

Btw, is there a reason you have commited the generated docs? The documentation setup seems a bit unconventional -- is there a reason for that?

**Comment by FedeClaudi** [comment]
> The documentation setup seems a bit unconventional -- is there a reason for that?

Yeah, that I'm just learning how to generate the docs with Documenter!

> Btw, is there a reason you have commited the generated docs?

What does this mean? How else would they show up on the gh pages?

**Comment by fredrikekre** [comment]
I meant why they are committed to the `master` branch.

> How else would they show up on the gh pages?

Documenter does it for you: https://github.com/FedeClaudi/Term.jl/blob/9be8e2fd3d5abb211851a7701896daec12f6d36e/docs/make.jl#L55

File: docs/src/adv/logging.md [hunk 2]
```diff
@@ -17,40 +17,41 @@ import Term: install_term_logger
...
+import Term: TermLogger, theme; import Logging: with_logger # hide
+with_logger(TermLogger(stderr, theme)) do # hide
+
 @info "My log message" 1+1 n="a string!" :x
+
+end # hide
 ```
 
 As you can see, it shows all the information that is normally there, an more! 
 If your log macro is called from within a function, it will also tell you that:
```
Comment (fredrikekre) [inline docs/src/adv/logging.md:32]: Btw, I noticed that this doesn't seem to work as in your screenshot (not outside of Documenter either).

**Pull request merged**